### ISSUE TITLE
Rotation Control Button

### ISSUE THREAD

**Issue by maschad96** [wnolfm/a11y-slider#9]
## General Description

-	The carousel contains a rotation control button that can stop and start automatic rotation.
	-	The rotation control button is the first element in the screen reader reading order.
	-	The rotation control button is always visible so it is available to all users whether they are interacting via a mouse, keyboard, assistive technology, or touch.
	-	If the carousel is rotating, its accessible name is Stop Automatic Slide Show, informing screen reader users that the slides are changing in addition to providing a way to stop the changes.
	-	If the carousel is not rotating, the accessible name of the button is Start Automatic Slide Show.
	-	If a user has activated the button to stop the show, the rotation will only restart if the button is activated. Moving keyboard focus or the mouse out of the carousel will not restart rotation.
	-	If keyboard focus is inside the carousel, or if the mouse is hovering over the carousel, the button is disabled; it cannot be used to start rotation.

## Keyboard Support
| Key | Function |
| --- | --- |
| tab | - Moves focus through interactive elements in the carousel. <br>- Rotation control, previous slide, and next slide buttons precede the slide content in the Tab sequence. |
| Enter or Space | Toggle the auto rotation of slides in the carousel.|
